### PR TITLE
History

### PR DESCRIPTION
**Pull request by kevinforrestconnors**
This pull request:
## **Templates**
- Removes `application.html*haml` and `login.html.haml`
  - Adds `application.html.erb` and `login.html.erb`in their place
- Adds `_header.html.erb` partial which is imported by `application.html.erb` and `login.html.erb`
- Adds a `index.html.erb` file in /texts/ directory with a form (styles not yet added) that should fetch the current user's data
## **Styles**
- Changes global styles slightly such as adding viewport handling
- Adds `.header` styles

### CODE REVIEW

File: app/models/receipt.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  attr_accessible :message, :student
+  # TODO: Add message status enumerated type
+  #  Unsent/Queued/Sent/Delivered/Errored
+  belongs_to :message
+  belongs_to :student
+
+  # Send message to student
+  def send()
+    # TODO: Check status and only send if not already sent
+    account_sid = 'ACc3ff9be899397461c075ffcf9e70f35a'
```
Comment (alexthemitchell) [inline app/models/receipt.rb:11]: This should not be in the public repository.

File: app/views/layouts/login.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <%= javascript_include_tag 'application' %> 
+    <%= csrf_meta_tags %>
+  <head>
+  <body>
+
+    <%= render '/shared/header' %>
+
+    <main>
+
+<!-- idk how to do this in ERB rn      <% if flash[:notice] %>
```
Comment (alexthemitchell) [inline app/views/layouts/login.html.erb]: Convert to `TODO:`

File: app/views/text/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<form>
+    <div class="recipients">
+
+        <h2>Select Recipients</h2>
+
+        <% if @current_user.classroom_ids.length > 1 %>
```
Comment (alexthemitchell) [inline app/views/text/_form.html.erb]: Add message for teachers with no classrooms

File: app/models/receipt.rb (hunk 1 above)
Comment (kevinforrestconnors) [inline app/models/receipt.rb]: Done

File: app/views/layouts/login.html.erb (hunk 2 above)
Comment (kevinforrestconnors) [inline app/views/layouts/login.html.erb]: done

**Pull request merged**